internal/services/accountServiceImpl.go

```diff
@@ -0,0 +1,81 @@
+package user
```

Comment:
Same would apply here. If this is just a single service and you would like to define interface and implementation, interface should go to the same file. Having file named Impl does not really provide any abstraction or value to it. Just above the service, you define interface you will be implementing and the file `account.go` should be enough.
The best practice is not to repeat stuff. That is common in java for example, but in go, /services/account.go is simply enough for anyone reading it knowing that this is a service.